src/query_bed.py

```diff
@@ -30,5 +30,20 @@ def main() -> None:
     # FIXME: put your code here
 
 
+    bed_table = Table() ## process bed file (input)
+    for line in args.bed:
+        bed_table.add_line(parse_line(line))
+
+    for line in args.query:
+        value_query = line.split()
+        new_table = bed_table.get_chrom(value_query[0])
+        if len(new_table) > 0:
```

Comment:
Why do you test whether the list is empty here? Would it change anything if you just looped through the empty list?